HISTORY.md

```diff
@@ -8,12 +8,14 @@
 ## Breaking behavior
 
 ## New features
+* Add support for serialization of config objects [#2164](https://github.com/TileDB-Inc/TileDB/pull/2164)
 * supporting serialization (using capnproto) build on windows [#2100](https://github.com/TileDB-Inc/TileDB/pull/2100)
```

Comment:
Not the biggest of deals but the lowercase 'supporting' has irked me for a few days.  Can you maybe uppercase to 'Supporting' while this one is still open?

Reply:
I'll address this in a separate PR.